packages/evolution-common/src/utils/DateUtils.ts

```diff
@@ -4,11 +4,16 @@
  * This file is licensed under the MIT License.
  * License text available at https://opensource.org/licenses/MIT
  */
-/**
- * This file provides a router with the routes to validate interviews
- * */
 
-// can return invalid dates: use isNaN on date.getTime() to check if it's valid
+import { Optional } from '../types/Optional.type';
+
+/**
+ * can return invalid dates: use isNaN on date.getTime() to check if it's valid
+ * @deprecated use parseDate instead
```

Comment:
Tu ajoutes une fonction déjà deprecated qui dit de s'utiliser elle-même à la place?

D'ailleurs c'est pas clair d'après la doc ce que cette fonction fait

Reply:
added doc and changed to correct use instead